client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -24,7 +24,7 @@ import { styled, getCountryPostalCodeSupport } from '@automattic/wpcom-checkout'
 import { useSelect, useDispatch } from '@wordpress/data';
 import debugFactory from 'debug';
 import { useTranslate } from 'i18n-calypso';
-import { useState, useCallback } from 'react';
+import { useState, useCallback, HTMLAttributes } from 'react';
```

Comment:
*Really minor* but if you use `import type` for `HTMLAttributes` on a line by itself, then the TS compiler can strip that import out during compilation. But this is fine either way.

Reply:
I appreciate all the helpful notes, there is always so much to learn! 😮